programs/main.cpp

```diff
@@ -158,6 +154,18 @@ std::pair<const char *, MainFunc> clickhouse_applications[] =
 #endif
 };
 
+/// Add an item here to register a new short name
+std::pair<std::string_view, std::string_view> clickhouse_short_names[] =
+{
+#if ENABLE_CLICKHOUSE_LOCAL
+    {"ch", "local"},
```

Comment:
I thought that this is an alias for `clickhouse`, so that you can use `ch client` (and so on). What do you think about this?

Reply:
P.S. personally I use such an alias (`alias ch=clickhouse`) locally (and I guess I'm not the only one)

Reply:
Good idea, let's do it, but let's also keep `ch` without arguments to run clickhouse-local.

Reply:
>but let's also keep ch without arguments to run clickhouse-local

I like this idea (actually I think it make sense for `clickhouse` as well, image user who do not know about `clickhouse-local` will mistype and see the clickhouse interpreter)